src/test/java/com/easyrest/tests/adminTests/SignInTests.java

```diff
@@ -0,0 +1,66 @@
+package com.easyrest.tests.adminTests;
+
+import com.easyrest.config.ConfigProvider;
+import com.easyrest.constants.Constants;
+import com.easyrest.facade.SignInFacade;
+import com.easyrest.pages.SignInPage;
+import com.easyrest.tests.BaseTest;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class SignInTests extends BaseTest {
+
+    private final String adminEmail = ConfigProvider.adminEmail;
+    private final String adminPassword = ConfigProvider.adminPassword;
+    private final String fakeEmail = ConfigProvider.fakeEmail;
+    private final String fakePassword = ConfigProvider.fakePassword;
+
+    private SignInPage signInPage;
+    private SignInFacade signInFacade;
+    private WebDriverWait wait;
+
+    @BeforeMethod
+    public SignInPage goToSignInPage() {
```

Comment:
Why do you need to return the SignInPage object?

Reply:
> Why do you need to return the SignInPage object?

Do not really need because I already have object of signInPage, and there is no need create one more. Fix it.